### PR TITLE
Add new auth service to take over Panda-issuing role from kahuna

### PR DESCRIPTION
**Pull request by theefer**
This PR just adds a new auth microservice. It's not being used yet, and the next stage will be to change kahuna to auth via the auth service instead of kahuna.

Needs to be built and artifacted before https://github.com/guardian/grid-infra/pull/118 (CF change) can be applied.

### CODE REVIEW

**Comment by kenoir** [comment]
:+1:

File: auth/app/lib/Config.scala [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package lib
+
+import com.gu.mediaservice.lib.config.{Properties, CommonPlayAppConfig, CommonPlayAppProperties}
+import com.amazonaws.auth.{BasicAWSCredentials, AWSCredentials}
+
+object Config extends CommonPlayAppConfig with CommonPlayAppProperties {
+
+  val appName = "auth"
+
+  val properties = Properties.fromPath("/etc/gu/auth.properties")
```
Comment (akash1810) [inline auth/app/lib/Config.scala]: do you need `panda.properties` too?

Comment (akash1810) [inline auth/app/lib/Config.scala]: meep, nevermind, its pulled in via the `PanDomainAuthActions` trait :smile:

**Comment by akash1810** [comment]
:+1:

**Comment by akash1810** [comment]
Just realised there's no update to the docker-compose file :cry:

Can add it after https://github.com/guardian/grid/pull/1730 gets merged.

**Pull request merged**